ui/pages/settings/advanced-tab/advanced-tab.component.test.js

```diff
@@ -21,6 +25,43 @@ jest.mock('../../../store/actions.ts', () => {
   };
 });
 
+jest.mock('webextension-polyfill', () => ({
+  runtime: {
+    getPlatformInfo: jest.fn().mockResolvedValue('mac'),
+  },
+}));
+
+jest.mock('../../../helpers/utils/export-utils', () => ({
+  exportAsFile: jest.fn(),
+}));
+
+// jest.mock('../../../ducks/app/app.ts', () => {
+//   return {
+//     displayErrorInSettings: () => mockDisplayErrorInSettings(),
+//     hideErrorInSettings: () => {
+//       return mockHideErrorInSettings();
+//     }
+//   };
+// });
+
+// jest.mock('../../../helpers/utils/export-utils', () => ({
+//   exportAsFile: () => mockExportAsFile
+// }));
+
+// jest.mock('../../../helpers/utils/export-utils', () => {
+//   return {
+//     exportAsFile: () => mockExportAsFile,
+//   };
+// });
+// jest.mock('../../../helpers/utils/export-utils');
```

Comment:
Should these commented mocks stay or go?

Reply:
File is updated now